suncalc3/README-HORIZON-CARD.md

```diff
@@ -0,0 +1,25 @@
+# Local copy of suncalc3
+
+This is a copy of module suncalc3 v2.0.5 with some minor modifications that bypass its wrong assumptions about time zones.
```

Comment:
Seems like an extreme step. Did you raise those wrong assumptions, do you see a path to remove this local copy at a later state?

Reply:
"its incorrect assumptions", or "imprecise", sounds better than "wrong". 😉
"some minor modifications to improve its time zone handling."?

Reply:
@ThomDietrich, extreme or not it was very necessary. When I find the time I might submit a PR to the original project but I really needed it to work in an "improved" way :)

@edwardtfn, wrong is wrong whether you call it that or "incorrect". But I get your point so I changed it to "improve its time zone handling".